openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts

```diff
@@ -1467,4 +1467,50 @@ Since we are introducing the Auto Classification workflow, we are going to remov
       [`Service Spec for the Ingestion Framework`]: `This impacts users who maintain their own connectors for the ingestion framework that are NOT part of the OpenMetadata python library (openmetadata-ingestion). Introducing the "connector specifcication class (ServiceSpec)". The ServiceSpec class serves as the entrypoint for the connector and holds the references for the classes that will be used to ingest and process the metadata from the source. You can see postgres for an implementation example.`,
     },
   },
+  {
```

Comment:
@aniketkatkar97 Please update version id [here](https://github.com/open-metadata/OpenMetadata/blob/ea205d3ff4c6ede659059fca5447cd808f37d93c/openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/whatsNewData.ts#L20)

Reply:
@chirag-madlani Thanks for reminding 👍 
Updated can you please review it again?